### DISCUSSION TITLE
[Feature] Should we document what folders are the actual original photo / video files?

### DISCUSSION THREAD

**Discussion by twitsforbrains** [immich-app/immich#2009] [Feature Request]
### The feature

I am using immich as a backup but just like the docs says, I don't want to use immich as my only backup.

Instead I want to set up a cron job that saves any new files to S3 so that if my laptop is lost, I can still retrieve my photos somehow. 

The problem is, I see files in the year / month folders and also in the encoded-video and original folders. The 'year / month'  folders seem to hold my photos so I should definitely save those to s3. However is it also necessary to save the original folder files? I see a subset of my live photos and videos in that folder but its unclear if those are also available in my year / month folders...And I presume it is safe to NOT store the encoded video folder? I'm assuming the originals are in the year / month folders?

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
When uploading a file to Immich, it is initially stored in the `original` folder. During processing, it gets moved into one of the year folders. If you've got an old install some files may not have been moved over, which would explain why they're still in there. Either way, you'll want to backup the year folders and the `original` folders. `encoded-video` and `thumb` are indeed safe to discard.

At some point we want to split the main library and Immich's own files (thumbs/encoded videos) into two separate folders, so the separation is more obvious.
